src/visualizers/VisualizerHistogram.ts

```diff
@@ -0,0 +1,211 @@
+import {VisualizerExportModule} from '@/visualizers/VisualizerInterface'
+import {VisualizerDefault} from './VisualizerDefault'
+import {natlog} from '../shared/math'
+
+class HistogramVisualizer extends VisualizerDefault {
+    name = 'Histogram'
+
+    binSize = 1
+    terms = 10
+    firstIndex = 1
+    linear = false
+
+    params = {
+        binSize: {
+            value: this.binSize,
+            forceType: 'integer',
+            displayName: 'Bin Size',
+            required: true,
+        },
+
+        firstIndex: {
+            value: this.firstIndex,
+            forceType: 'integer',
+            displayName: 'First Index',
+            required: true,
+        },
+
+        terms: {
+            value: this.terms,
+            forceType: 'integer',
+            displayName: 'How many terms of the series',
+            required: true,
+        },
+    }
+
+    checkParameters() {
+        const status = super.checkParameters()
+
+        if (this.params.binSize.value < 1) {
+            status.isValid = false
+            status.errors.push('Bin Size can not be less than 1')
+        }
+
+        if (this.params.binSize.value < 1) {
+            status.isValid = false
+            status.errors.push('Bin Size can not be less than 1')
+        }
+
+        if (this.params.firstIndex.value < 1) {
+            status.isValid = false
+            status.errors.push('First index can not be less than 1')
```

Comment:
Minor nitpick: in the other checks, you have capital "Bin Size", but in this one you have "First index". The display name is "First Index", so perhaps it should be capitalized accordingly?

Reply:
Addressed in https://github.com/numberscope/frontscope/pull/183/commits/6aa8c435ee687c75c471fc7e539db78936818362. @Decos14, mark this as resolved if you're okay with this change. Otherwise, let me know and I can change it.

Reply:
Resolving this conversation per @Decos14's comment in https://github.com/numberscope/frontscope/pull/183#issuecomment-1307874402.